src/constants/color.js

```diff
@@ -25,12 +25,14 @@ export const airportOutline = "hsl(250, 41%, 79%)";
 export const airportRunway = "hsl(250, 41%, 79%)";
 export const airportLabel = "hsl(250, 71%, 29%)";
 
+//TODO - rename this variable to "palette"
 export const shields = {
   black: "black",
   blue: "#003f87", // Pantone 294
   brown: "#693f23", // Pantone 469
   green: "#006747", // Pantone 342
   orange: "#f38f00", // Pantone 152
+  strong_orange: "#d45d00", // Pantone 717 C
```

Comment:
For what it’s worth, #576 uses `#d93f27`, which comes from the California MUTCD, so it’s designed to be used in conjunction with the other colors in this palette.

Reply:
Screen shot with the CA MUTCD vivid orange labels but with the current color icons:

![image](https://user-images.githubusercontent.com/3254090/211616599-defcf8e1-e3d1-4b35-89f9-70933cb53d3d.png)

I think the California orange doesn't have enough contrast and we want a darker hue?

Reply:
You’re right, though I guess that means this burnt-orange color choice will end up being specific to labels (or icons color-coordinated with them) rather than part of a style-wide color palette that everything adheres to.

Reply:
Here's where I am with the burn orange:
![image](https://user-images.githubusercontent.com/3254090/211675528-86395b4c-bb77-4fe6-9a5e-1cc05eabd01a.png)

Should we explore other colors in the palette or try to tweak this or is this good enough?  I also envision that brown will be used for recreational things and so I'm wary about picking a color too close to that.

Reply:
Burnt orange [as my alma mater defines it](https://brand.utexas.edu/identity/color/) is `#bf5700`. I tried it out, and it's still recognizably orange, but a little darker and arguably more legible:

![Screenshot from 2023-01-10 18-21-20](https://user-images.githubusercontent.com/1732117/211682759-1d29acce-6c14-4ddd-880f-066adae8b60a.png)

Reply:
You got me craving a Texas navel orange, burnt.

I think any brown color we use as a foreground color would end up considerably darker, so I’m not too concerned with a collision.

Reply:
For quality assurance, I did an in-person survey and found this flavor of orange to work just fine:
![image](https://user-images.githubusercontent.com/3254090/211709592-c9453c44-205a-457d-8a83-01b41ac66ad0.png)